src/public/js_native_ext_api.h

```diff
@@ -28,13 +28,13 @@ typedef struct napi_ext_env_scope__ *napi_ext_env_scope;
 typedef struct napi_ext_ref__ *napi_ext_ref;
 
 // A callback to return buffer synchronously
-typedef void (*napi_ext_buffer_callback)(napi_env env, uint8_t const *buffer, size_t buffer_length, void *buffer_hint);
+typedef void (__cdecl *napi_ext_buffer_callback)(napi_env env, uint8_t const *buffer, size_t buffer_length, void *buffer_hint);
```

Comment:
>__cdecl [](http://example.com/codeflow?start=14&length=7)

Can we add some tests that verify that we don't lose this signature over time?

Reply:
I cannot think about any such test. Any ideas?